repl/src/test/resources/scriptLevelCaching/predef.scala

```diff
@@ -0,0 +1,42 @@
+import ammonite.repl.frontend.ReplBridge.repl
```

Comment:
This file should not be necessary; if you need this code-string, you can read it off of `ammonite.repl.Main.defaultPredefString`. We shouldn't copy-paste things